src/katgpucbf/xbgpu/engine.py

```diff
@@ -443,9 +444,19 @@ def __init__(
             tx_enabled=self._init_tx_enabled,
         )
 
-    @staticmethod
-    def populate_sensors(sensors: aiokatcp.SensorSet) -> None:
+    def populate_sensors(self, sensors: aiokatcp.SensorSet) -> None:
         """Define the sensors for an XBEngine."""
+        # Static sensors
+        sensors.add(
+            aiokatcp.Sensor(
+                str,
+                "chan-range",
+                "The range of channels processed by this XB-engine",
```

Comment:
```suggestion
                "The range of channels processed by this XB-engine, inclusive",
```

To be clear that the end channel is included (feel free to phrase it differently - this is what corr2 did)